run_dir/static/js/lanes_ordered.js

```diff
@@ -0,0 +1,284 @@
+const vLanesOrderedMain = ({
+    data() {
+        return {
+            statistics_data: {},
+            in_focus: [null, null, null]
+        }
+    },
+    methods: {
+        fetchStatistics(key1, key2, key3) {
+            let url = '/api/v1/lanes_ordered'
+            /* Build the url, where only defined keys should be passed */
+            let my_arguments = [key1, key2, key3];
+            for (var i = 0; i < my_arguments.length; i++) {
```

Comment:
```suggestion
            for (var i = 0; i < my_arguments.length; i++) {
```
Any reason to declare i as var here?

Reply:
Not sure, I guess `let` would be the alternative, or something else?